setup/lang/en/default.inc.php

```diff
@@ -186,7 +186,7 @@
 $_lang['password_err_invchars'] = 'Your password may not contain any invalid characters, such as /, \\, &apos;, &quot;, (, ) or {}.';
 $_lang['password_err_nomatch'] = 'Does not match password';
 $_lang['password_err_ns'] = 'Password is empty';
-$_lang['password_err_short'] = 'Your password must be at least 6 characters long.';
+$_lang['password_err_short'] = 'Your password must be at least 8 characters long.';
```

Comment:
Maybe the 8 could be replaced by a placeholder [[+length]] and set that value in the lexicon call.